packages/payment-integration-api/src/common/types/webpack.d.ts

```diff
@@ -0,0 +1 @@
+declare const LIBRARY_VERSION: string;
```

Comment:
Do we need to move this declaration here?

Reply:
This declaration needed to remove IDE errors in places where LIBRARY_VERSION in use

Reply:
We should have this declaration here because LIBRARY_VERSION was duplicated into payment-integration-api project to avoid using same variable in other packages from core package